packages/venia-concept/src/components/Checkout/braintreeDropin.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @fileoverview This component uses the BrainTree Web Drop In to
+ * hook into Web Payments and the Payment Request API to
+ * submit payments via BrainTree.
+ *
+ * See:
+ * https://github.com/braintree/braintree-web-drop-in
+ * https://braintree.github.io/braintree-web-drop-in/docs/current/index.html
+ * https://developers.braintreepayments.com/guides/drop-in/overview/javascript/v3
+ */
+
+import React, { Component } from 'react';
+import { bool, func } from 'prop-types';
+import dropin from 'braintree-web-drop-in';
+
+const BRAINTREE_SANDBOX_TOKEN = 'sandbox_8yrzsvtm_s2bg8fs563crhqzk';
```

Comment:
Since venia is a concept/reference perhaps this sort of thing should live in a top-level config/keys file with a big header like `EDIT THESE TOKENS WITH YOUR OWN DEETS`?

Alternatively if this is a one-off, maybe just add a comment :P

Reply:
Great call, I'll move this to `.env`.

Reply:
Moved in https://github.com/magento-research/pwa-studio/pull/744/commits/6b85da632da0cecf05793b4b0e5e7dc4b49ffdd9.